.circleci/config.yml

```diff
@@ -13,6 +13,11 @@ jobs:
     working_directory: ~/circleci-f36-build
     steps:
       - checkout
+      - vault/get-secrets:
+          template-preset: packages-read
+      - run: |
+          echo "//npm.pkg.github.com/:_authToken=${GITHUB_PACKAGES_READ_TOKEN}" > ~/.npmrc
+          echo "@contentful:registry=https://npm.pkg.github.com" >> ~/.npmrc
```

Comment:
Do we need this? We never read from our private registry.

Reply:
Without this, it was failing to install the browserlist:
https://app.circleci.com/pipelines/github/contentful/forma-36/11276/workflows/94cfd9cb-ea24-4742-a631-3d8f116eb414/jobs/31390

Probably because all our packages on github are marked as internal visibility